aip/0008.md

```diff
@@ -172,6 +172,15 @@ When AIPs reference other AIPs, the prosaic text **must** use the format
 link to the relevant AIP. AIP links **may** point to a particular section of
 the AIP if appropriate.
 
+**Important:** AIP links **must** use the relative path to the file in the
+repository (such as `./0008.md` for core AIPs, or `../0008.md` for AIPs in a
```

Comment:
Couldn't this also be a full link? e.g., `aip.dev/8` ?

Reply:
No. It breaks running locally, and it ships you improperly when navigating GitHub markdown.

Reply:
Gotcha. Let's clarify that below.